db/db.js

```diff
@@ -46,8 +46,139 @@ const deleteExample = (request, response) => {
+    pool.query("SELECT id From Products WHERE id = $1;", [id], (err, res) => {
+        if (err) {
+            throw err;
+        } else {
+            if (res.rowCount === 0) {
+                addTheItem(id, name, price, description);
+            }
+        }
+    });
+};
+
+//Adds the item to the table Products and sends it on to addToDishes()
+const addTheItem = (id, name, price, description) => {
+    pool.query(
+        "INSERT INTO Products VALUES($1, $2::text, $3, $4);",
+        [id, name, price, true], //todo change to actual status on item
+        (err, res) => {
+            if (err) {
+                throw err;
+            } else {
+                console.log(
+                    "amount of rows inserted into products:",
+                    res.rowCount
+                );
+                addToDishes(id, description);
+            }
+        }
+    );
+};
+
+//Adds item to dishes
+const addToDishes = (id, description) => {
+    pool.query(
+        "INSERT INTO Dishes VALUES($1, $2, $3);",
+        [id, "category", description],
+        (err, res) => {
+            if (err) {
+                throw err;
+            } else {
+                console.log(
+                    "amount of rows inserted into dishes:",
+                    res.rowCount
+                );
+            }
+        }
+    );
+};
+
+//Deletes menu item based on id
+const deleteMenuItemBasedOnID = (id) => {
+    pool.query(
+        "DELETE FROM OrdersAndProducts WHERE orderedProduct = $1;",
+        [id],
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+        }
+    );
+    pool.query(
+        "DELETE FROM OrderedWith WHERE orderedProduct = $1;",
+        [id],
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+        }
+    );
+
+    pool.query(
+        "DELETE FROM OrderedProducts WHERE dish = $1;",
+        [id],
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+        }
+    );
+    pool.query(
+        "DELETE FROM AvailableToppings WHERE dish = $1;",
+        [id],
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+        }
+    );
+    pool.query("DELETE FROM Dishes WHERE id = $1;", [id], (error, results) => {
+        if (error) {
+            throw error;
+        }
+    });
+    pool.query(
+        "DELETE FROM Products WHERE id = $1;",
+        [id],
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+        }
+    );
+};
+
+//get the id of the last added product (the highest number)
+function getLastMenuItemID(callback) {
+    pool.query(
+        "SELECT id FROM Products ORDER BY id DESC LIMIT 1",
+        (error, results) => {
+            if (error) {
+                throw error;
+            }
+            callback(results.rows);
```

Comment:
Same as before, for consistency I think it should be:
```suggestion
            } else {            
                callback(results.rows);
            }
```